src/stories/Components.Badge.stories.tsx

```diff
@@ -0,0 +1,49 @@
+// Button.stories.ts|tsx
+
+import type { Meta, StoryObj } from "@storybook/react";
+import { Badge } from "../components/Badge/Badge";
+import { HabitatTheme } from "../utils/theme.css";
+import { BadgeIcon } from "../assets/badge/BadgeIcon";
```

Comment:
the badge icon is used only in this test, can we inline it in this file?

Reply:
Good point, i'll move the BadgeIcon inside the stories component because is use only to show the possibility to insert an icon inside the component.